charm/src/charm.ts

```diff
@@ -360,7 +364,7 @@ export class CharmManager {
   }
 
   // FIXME(JA): this really really really needs to be revisited
-  async syncRecipe(charm: Cell<Charm>): Promise<string | undefined> {
+  async syncRecipe(charm: Cell<Charm>): Promise<string> {
```

Comment:
Can you say more about the future improvements we need here?

Reply:
We should clarify what recipes go where - since right now we are throwing everything at blobby.  should we rely on blobby for anything more than "spellbook/sharing"

how does that impact spellcasting? maybe everything should go?

would love to whiteboard about this